build/kube_storage_monitor/node-watcher/README.md

```diff
@@ -0,0 +1,152 @@
+# Node watcher
+
+Below is how to deploy node watcher to watch node failure conditions
+
+## Deployment
+
+### Create a kubernetes cluster containing two nodes(192.168.0.101, 192.168.0.105)
+``` kubectl get nodes
+NAME            STATUS     ROLES     AGE       VERSION
+192.168.0.101   Ready      <none>    8h        v1.10.0-alpha.1.5216+9b7439d77d0b29
+192.168.0.105   NotReady   <none>    30m       v1.10.0-alpha.1.1689+77d6ca8d4bb0b1-dirty
```

Comment:
This is already `NotReady`?

Reply:
Yes, we need to check that when node breaks down(NotReady for a period of time or Terminated), local PVs on that node are marked or not.